piccolo/apps/playground/commands/run.py

```diff
@@ -161,6 +163,39 @@ def get_readable(cls) -> Readable:
         )
 
 
+class Movie(Table):
+    id: Serial
+    name = Varchar()
+    genres = M2M(LazyTableReference("GenreToMovie", module_path=__name__))
+
+    @classmethod
+    def get_readable(cls) -> Readable:
+        return Readable(
+            template="%s",
+            columns=[cls.name],
+        )
+
+
+class Genre(Table):
+    id: Serial
+    name = Varchar()
+    movies = M2M(LazyTableReference("GenreToMovie", module_path=__name__))
+
+    @classmethod
+    def get_readable(cls) -> Readable:
+        return Readable(
+            template="%s",
+            columns=[cls.name],
+        )
+
+
+class GenreToMovie(Table):
+    id: Serial
+    movie = ForeignKey(Movie)
+    genre = ForeignKey(Genre)
+    reason = Text(help_text="For testing additional columns on join tables.")
```

Comment:
I would change this to `GenreToBand`, as we already have a `Band` table.

Or `GenreToAlbum`, whichever makes more sense.

Reply:
I'll use `GenreToBand` because as you say the `Band` table is already here. Thanks